lib/Analysis/Memory/GlobalLiveMemory.cpp

```diff
@@ -319,6 +319,19 @@ bool GlobalLiveMemory::runOnModule(Module &M) {
     LS->setOut(MayLives);
     LiveDFFwk LiveFwk(IntraLiveInfo, DefInfo, DT);
     solveDataFlowDownward(&LiveFwk, TopRegion);
+    auto ExpandCollapsed = [](LiveSet *LS) {

```

Comment:
```suggestion
    auto ExpandCollapsed = [](LiveSet &LS) {
```
Для безопасности, чтобы не забыть, что там был `std::unique_ptr`.